torch/linalg/__init__.py

```diff
@@ -830,35 +880,36 @@
 """)
 
 vector_norm = _add_docstr(_linalg.linalg_vector_norm, r"""
-linalg.vector_norm(input, ord=None, dim=None, keepdim=False, *, dtype=None, out=None) -> Tensor
+linalg.vector_norm(A, ord=None, dim=None, keepdim=False, *, dtype=None, out=None) -> Tensor
+
+Computes the vector norm of a vector.
 
-Computes the vector norm of a vector. Supports batched inputs.
+If :attr:`A` is complex valued, it computes the norm of :attr:`A`\ `.abs()`
+The returned tensor will always be real-valued, even when :attr:`A` is complex.
 
-If :attr:`input` is complex valued, it computes the norm of :attr:`input`\ `.abs()`
+Supports inputs of float, double, cfloat and cdouble dtypes.
+Also supports batched inputs, and, if the input is batched, the output is batched with the same dimensions.
 
-- If :attr:`dim`\ `= None`, the norm of the flattened :attr:`input` will be computed.
+- If :attr:`dim`\ `= None`, the norm of the flattened :attr:`A` will be computed.
```

Comment:
"the norm of the flattened :attr:\`A\` will be computed" -> ":attr:\`A\` will be flattened before a norm is computed"

Reply:
Corrected here and in `norm`.